openlibrary/conftest.py

```diff
@@ -67,3 +67,18 @@ def render(name, *a, **kw):
         d = infobase_render_template(name, *a, **kw)
         return six.text_type(d) if as_string else d
     return render
+
+
+def pytest_addoption(parser):
```

Comment:
This plus the shell script feel like unnecessary complexity, a `@parametrize` decorator to  `test_valid_template()` could take the list of filenames. I think the important part here is to ensure the list of files tested are the same as those that get compiled for real. Ideally the test would get the list from the same source.